internal/sync/configs/ovnk-config.yaml

```
@@ -0,0 +1,26 @@
+  repo_link: "https://github.com/ovn-org/ovn-kubernetes"
+  branch: master
+  remote_api_directory: go-controller/pkg/crd/routeadvertisements
+  local_api_directory: schemes/ovn/routeadvertisement
+  replace_imports:
+    - old: '"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/crd/types"'
+      new: '"github.com/rh-ecosystem-edge/eco-goinfra/pkg/schemes/ovn/types"'
+  excludes:
+    - "*_test.go"
+    - "*_webhook.go"
+    - "testdata"
+...
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)